frontend/src/shared/layout/navigation/side-navigation.tsx

```diff
@@ -178,18 +183,19 @@ export default function SideNavigation () {
                 }
             />
             {window.gitInfo && (
-                <Box>
-                    <div
-                        style={{
-                            position: 'absolute',
-                            bottom: 10,
-                            left: 10,
-                            fontSize: '10px',
-                            color: '#969696'
-                        }}
-                        title={window.gitInfo?.gitHash}>{window.gitInfo?.revisionTag}</div>
-                </Box>
+                <div
+                    style={{
+                        fontSize: '10px',
+                        marginTop: 'auto',
+                        color: '#969696',
```

Comment:
I know we have theme-able styling now. Just want to confirm this color should be hard coded as opposed to something more dynamic. I think @LightSeekerSC did the them stuff so perhaps he can weigh in?

Reply:
This color was already there but I can change it to anything that is more appropriate. Not related to my current changes though.

Reply:
From a theming perspective we should be able to just ditch the color all together - it's going to default to the correct color based on the theme.